src/public/services/PublicFormService.ts

```diff
@@ -0,0 +1,78 @@
+import axios from 'axios'
+
+import {
+  EmailSubmissionDto,
+  EncryptSubmissionDto,
+  SubmissionResponseDto,
+} from '../../types/api'
+import { createEmailSubmissionFormData } from '../utils/submission'
+
+const PUBLIC_FORMS_ENDPOINT = '/api/v3/forms'
+
+/**
+ * Submits an email mode form submission.
+ * @param formId id of form to submit submission for
+ * @param content content of submission
+ * @param attachments any attachments included in submission
+ * @param captchaResponse string if captcha to be included, defaults to null
+ *
+ * @returns SubmissionResponseDto if successful, else SubmissionErrorDto on error
+ */
+export const submitEmailModeFormSubmission = async ({
```

Comment:
the function name is a bit clunky because it has both `submit` and `submission` - would `submitEmailModeForm` convey the same idea?

Reply:
gotcha

Reply:
resolved in 5ff56b5